examples/rh-sso/README.md

```diff
@@ -0,0 +1,112 @@
+# Running APIcast with RH-SSO and OIDC for API Authentication
+
+This example shows you how to use Red Hat Single Sign-On to verify the identity of an [End-User](https://openid.net/specs/openid-connect-core-1_0.html#Terminology) and issue access tokens for use by clients wishing to access the End-User resources. 3scale will add an API Management layer, which includes access control, rate-limiting policies, and also a self-service model of registering the client applications for consuming the API.
+
+## Pre-requisites
+
+- 3scale account - You can sign up for a Free Trial [here](https://www.3scale.net/signup/)
+- Red Hat SSO instance - [Installation](https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.0/html-single/server_installation_and_configuration_guide/) and [Configuration](https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.0/html/getting_started_guide/) instructions
+- API - You can use the 3scale [echo API](https://echo-api.3scale.net:443)
+- Client Application to consume the API - You can use a client such as [Postman](https://www.getpostman.com/)
+- APIcast instance
+
+## Red Hat Single Sign-On Configuration for APIcast
+
+1. [Create a new realm](https://access.redhat.com/documentation/en-us/red_hat_single_sign-on/7.0/html/getting_started_guide/create_a_realm_and_user#create-realm) (different from Master)
```

Comment:
I'd say it makes sense to export the RHSSO configuration and add it to the example. 

From there it wouldn't be that hard to just have docker compose to start keycloak and have full working example with no external dependencies.